enterprise/server/remote_execution/containers/firecracker/BUILD

```diff
@@ -80,6 +80,9 @@ genrule(
     ],
 )
 
+# To remotely execute this test, a couple of tag_filters are needed:
+# bazel test --config=remote --test_tag_filters=+bare --build_tag_filters=+bare \
```

Comment:
I don't think the `build_tag_filters` are required? otherwise LGTM

Reply:
Yeah, looks like it works without that option. Removed.